app/celery/scheduled_tasks.py

```diff
@@ -222,6 +222,24 @@ def check_templated_letter_state():
             )
 
 
+@notify_celery.task(name="in-flight-to-inbox-sms")
+@statsd(namespace="tasks")
+def in_flight_to_inbox_sms():
+    _, notifications = sms_queue.expire_inflights()
+
+    for notification in notifications:
+        sms_queue.publish(notification)
+
+
+@notify_celery.task(name="in-flight-to-inbox-email")
+@statsd(namespace="tasks")
+def in_flight_to_inbox_email():
```

Comment:
I think we could get away with just one beat process, and not one per notification types (emails + sms). We just need to call the two queues `expire_inflights` function in the same beat function.